Student_BCOs/XiyingDing_BCO_000524.json

```diff
@@ -1,27 +1,79 @@
 {
     "bco_id": "http://biocomputeobject.org/BCO_000524",
-    "checksum": "893227528679a28274e102aa60ef67b64dbab5f8e064060edc5ba9f6fa4b464d",
+    "checksum": "18ba0128bf0e79405419828fb948b34b27c6db1effd0c06cf213635786a25070",
     "bco_spec_version": "1.3.0",
     "provenance_domain": {
-        "name": "TAP: a targeted clinical genomics pipeline for detecting transcript variants using RNA-seq data",
+        "name": "Detection of transcript variation by analysis of RNA-seq data using targeted clinical genomics pipeline TAP",
         "version": "0.4.2",
         "license": "http://creativecommons.org/licenses/by/4.0/",
         "created": "2019-10-07T22:19:18.384004",
-        "modified": "",
+        "modified": "2019-10-18T02:29:31.360979",
+        "review": [
+            {
+                "status": "in-review",
+                "reviewer_comment": "Reviewed by BioCompute technical assistant, approval pending modification by author",
+                "reviewer": {
+                    "orcid": "",
+                    "affiliation": "George Washington University",
+                    "contribution": [
+                        "curatedBy"
+                    ],
+                    "name": "XiyingDing",
+                    "email": "xzd@gwu.edu"
+                },
+                "date": "10/17/2019"
+            }
+        ],
```

Comment:
@xzd5071 
Please either remove the review section  from your BCO or remove the reviewer_comment from this section. If you'd like to discuss why tomorrow, please talk to @kee007ney 
Thank you.